dessia_common/workflow/blocks.py

```diff
@@ -963,6 +978,53 @@ def _to_script(self) -> ToScriptElement:
         return ToScriptElement(declaration=script, imports=[self.full_classname])
 
 
+class ConcatenateStrings(Block):
```

Comment:
In https://github.com/Dessia-tech/dessia_common/pull/392, position has been added to __init__ and to_script has been improved. 

If this PR is merged before the one just above, we will need to update 392 to handle position in init and in to_script. 
If not, we should update this PR one 392 is merged

Reply:
This is to be merged directly into master, so it will happen on a eventual chore/testing-to-dev branch